arangod/Cluster/ServerState.cpp

```diff
@@ -47,6 +48,28 @@
 
 using namespace arangodb;
 using namespace arangodb::basics;
+using namespace arangodb::consensus;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief helper function to conveniently assemble paths from string constants
+///
+///        this should really be moved to a more appropriate place, and maybe
+///        do checks and/or trims on the strings that are passed in.
+////////////////////////////////////////////////////////////////////////////////
+static std::string concatPath(std::vector<std::string> const& components) {
+  if (components.empty()) {
```

Comment:
the entire method can be replaced by `return StringUtils::join(components, '/');`

Reply:
Thanks, I did not check `StringUtils.h`, just the `.cpp` file and hence did not see this one.